find_ad_culprit.sh

```
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+echo "🔍 Deep Android Dependency Analysis for AD_ID Permission"
+echo "======================================================"
+
+cd android
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)